pages/common/assets/sass/style.scss

```diff
@@ -74,6 +74,10 @@ th {
   display: none;
 }
 
+.grey {
+  color: $grey-1;
+}
+
```

Comment:
Does this actually get used anywhere?

Reply:
it _did_

Reply:
In the designs there is a "Step 1 of 2" above the title on the password setup page, but as this is only one step now, I removed it